tools/scoring/tests/samples/solution/real_world_solution.yaml

```diff
@@ -0,0 +1,383 @@
+    phase1_phase2_line_voltage_sensor:
+      present_value: data.Volts_AB.present-value
+      units:
+        key: data.Volts_AB.present-value
+        values:
+          volts: volts
+    phase1_phase3_line_voltage_sensor:
+      present_value: data.Volts_AC.present-value
+      units:
+        key: data.Volts_AC.present-value
+        values:
+          volts: volts
+    phase1_power_sensor:
+      present_value: data.kW_A.present-value
+      units:
+        key: data.kW_A.present-value
+        values:
+          kilowatts: kilowatts
+    phase1_powerfactor_sensor:
+      present_value: data.PF_A.present-value
+      units:
+        key: data.PF_A.present-value
+        values:
+          no_units: no-units
+    phase2_line_current_sensor:
+      present_value: data.Current_B.present-value
+      units:
+        key: data.Current_B.present-value
+        values:
+          amperes: amperes
+    phase2_neutral_line_voltage_sensor:
+      present_value: data.Volts_BN.present-value
+      units:
+        key: data.Volts_BN.present-value
+        values:
+          volts: volts
+    phase2_phase3_line_voltage_sensor:
+      present_value: data.Volts_BC.present-value
+      units:
+        key: data.Volts_BC.present-value
+        values:
+          volts: volts
+    phase2_power_sensor:
+      present_value: data.kW_B.present-value
+      units:
+        key: data.kW_B.present-value
+        values:
+          kilowatts: kilowatts
+    phase2_powerfactor_sensor:
+      present_value: data.PF_B.present-value
+      units:
+        key: data.PF_B.present-value
+        values:
+          no_units: no-units
+    phase3_line_current_sensor:
+      present_value: data.Current_C.present-value
+      units:
+        key: data.Current_C.present-value
+        values:
+          amperes: amperes
+    phase3_neutral_line_voltage_sensor:
+      present_value: data.Volts_CN.present-value
+      units:
+        key: data.Volts_CN.present-value
+        values:
+          volts: volts
+    phase3_power_sensor:
+      present_value: data.kW_C.present-value
+      units:
+        key: data.kW_C.present-value
+        values:
+          kilowatts: kilowatts
+    phase3_powerfactor_sensor:
+      present_value: data.PF_C.present-value
+      units:
+        key: data.PF_C.present-value
+        values:
+          no_units: no-units
+    power_sensor:
+      present_value: data.kW.present-value
+      units:
+        key: data.kW.present-value
+        values:
+          kilowatts: kilowatts
+    powerfactor_sensor:
+      present_value: data.PF.present-value
+      units:
+        key: data.PF.present-value
+        values:
+          no_units: no-units
+    reactive_power_sensor:
+      present_value: data.kVAR.present-value
+      units:
+        key: data.kVAR.present-value
+        values:
+          kilovolt_amperes_reactive: kilo-volt-amperes-reactive
+  type: METERS/EM_PWM_AVCM_PLPM_PVCM
+  guid: a51ee956-0ea4-47ae-b398-b1b38eb103fa
+power-meter-EV_L1C:
```

Comment:
can you please leave empty lines between entities for visibility

Reply:
https://github.com/google/digitalbuildings/pull/555/commits/ff694d3aaa11f680e25c31a36a3cc9a6c94230a3